packages/web-react/src/components/TextArea/README.md

```diff
@@ -1,14 +1,34 @@
 # TextArea
 
-TextArea enables the user to add longer text to a form. It has textarea, label,
-and an optional message. It could be disabled or have a validation state. The label could be hidden
-and show if the textarea is required.
+TextArea enables the user to add longer text to a form.
+It could be disabled or have a validation state with validation text.
```

Comment:
```suggestion
It could be disabled or have a validation state with optional validation text.
```
It should be mentioned, that the validation text is optional.